app/views/gobierto_participation/processes/index.html.erb

```diff
@@ -4,20 +4,44 @@
   </strong>
 <% end %>
 
-<h1>Gobierto Participation processes index</h1>
+<%= content_for :body_attributes do %>
+  class="theme-participation gobierto_participation"
+<% end %>
 
-Processes:
+<div class="gradient-bg">
```

Comment:
![image](https://user-images.githubusercontent.com/9287468/30598169-54ca763a-9d59-11e7-9ea2-f097f253124c.png)

The second row is a little bit tabbed to the right.

Reply:
Fixed